runtime/src/iree/hal/drivers/local_sync/cts/CMakeLists.txt

```diff
@@ -28,11 +28,8 @@ if(IREE_HAL_EXECUTABLE_LOADER_EMBEDDED_ELF)
     DEPS
       iree::hal::drivers::local_sync::registration
     EXCLUDED_TESTS
-      # TODO(#4680): command buffer recording so that these can run on sync HAL
-      "command_buffer"
-      "event"
-      "semaphore_submission"
-  )
+      "semaphore_submission"  # SubmitWithWait hangs?
```

Comment:
We could switch from `EXCLUDED_TESTS` to `XFAIL_TESTS` using `WILL_FAIL`. I'd be concerned about tests failing due to timeouts (like `semaphore_submission` here), causing ctest to take a consistently long time on known-failing tests, but we could also just lower the timeouts from 60 seconds to 20 seconds or so?

https://cmake.org/cmake/help/latest/manual/cmake-properties.7.html#test-properties

Reply:
Oooooo there is also https://cmake.org/cmake/help/latest/prop_test/DISABLED.html we could use that in other places instead of just omitting tests entirely